flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
what's the reason?

Reply:
It fails to build https://flathub.org/builds/#/builders/51/builds/7093 and the builder gets stuck until timeout.

Reply:
but
```
-- Found OpenGL: /usr/lib/aarch64-linux-gnu/libOpenGL.so   
-- Could not find OpenGL ES 1.1 libraries
```